commands/command_fetch.go

```diff
@@ -99,7 +101,27 @@ func fetchCommand(cmd *cobra.Command, args []string) {
 		}
 	}
 
-	if len(args) > 1 {
+	if fetchUseStdin {
+		if len(args) > 1 {
+			Print(tr.Tr.Get("Further command line arguments are ignored with --stdin"))
+			os.Exit(1)
+		}
```

Comment:
Looking at the `git fetch` command's manual I noticed that it [permits](https://git-scm.com/docs/git-fetch#Documentation/git-fetch.txt---stdin) the `--stdin` option to be combined with refs specified on the command line, and I wondered if we should try to match that behaviour.

If we stick with this design for now, though, we could just use the `Exit()` function to output the warning message, like we do in several other instances below (e.g., for the `Cannot combine --all with --recent` message, etc.)